portfolio/src/main/java/com/google/sps/servlets/NumCommentsServlet.java

```diff
@@ -54,30 +56,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<Integer> numComments = new ArrayList<> ();
+    Map<Integer, Integer> numComments = new HashMap<Integer, Integer> ();
```

Comment:
nit: You can use `int` instead of `Integer` here, converting between int and Integer is called boxing (the other way is called unboxing) and can add execution overhead that isn't necessary here.